src/catalog/catalog_set.cpp

```diff
@@ -225,6 +257,10 @@ void CatalogSet::Undo(CatalogEntry *entry) {
 		// delete the entry from the dependency manager as well
 		catalog.dependency_manager->EraseObject(to_be_removed_node);
 	}
+	if (entry->name != to_be_removed_node->name) {
+		// rename: clean up the new name when the rename is rolled back
+		mapping.erase(mapping.find(to_be_removed_node->name));
```

Comment:
Dumb C++ stl question: why do we `erase` the iterator returned by `find`, instead of passing the key to `erase` directly?

Reply:
Good point, for unordered_maps we can just pass the key itself. Thanks!